src/swbf_fnv_hashes.cpp

```diff
@@ -1,31 +1,34 @@
 
 #include "swbf_fnv_hashes.hpp"
+#include "string_helpers.hpp"
 #include "synced_cout.hpp"
 
 #include <cstdint>
-#include <string>
+#include <fstream>
+#include <regex>
+#include <shared_mutex>
 #include <unordered_map>
 #include <utility>
 
 using namespace std::literals;
```

Comment:
Small style issue, there should be newline between this and the namespace below.
```suggestion
using namespace std::literals;

```